applications/fishing-map/src/features/search/Search.tsx

```diff
@@ -284,7 +284,6 @@ function Search() {
             <label className={styles.title}>{t('search.title', 'Search')}</label>
             <Choice
               options={searchOptions}
-              disabledTooltip={t('search.advancedDisabled')}
```

Comment:
We lost the tooltip when the option is disabled, don't we? So we should go to the [searchOptions](https://github.com/GlobalFishingWatch/frontend/blob/fishing-map/download-activity-popup/applications/fishing-map/src/features/search/Search.tsx#L77) and include the tooltip when the user is not logged